src/devices/Hmc5883l/MeasuringMode.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Hmc5883l
+{
+    /// <summary>
+    /// The mode of HMC5883L measuring
+    /// </summary>
+    public enum MeasuringMode
+    {
+        Continuous = 0x00,
```

Comment:
Can we add a bit more description to each measuring mode to understand which one should I use?